service/pkg/server/options.go

```diff
@@ -71,6 +72,21 @@ func WithAuthZDefaultPolicyExtension(policies [][]string) StartOptions {
 	}
 }
 
+// deprecated: Use WithAuthZPolicy
```

Comment:
Did you mean to put this on another option ?

Reply:
Yep good call!